src/svelte/src/components/DataMetrics/ByteFrequencyGraph.svelte

```diff
@@ -138,6 +138,7 @@ limitations under the License.
           const startOffsetTemp = parseInt(e.detail.value)
```

Comment:
the parseInt calls need to be done with respect to the address radix also.  For example `parseInt(e.detail.value, 16)` for hex, etc.

Reply:
Ah so we're not locking the offset to decimal?

Reply:
Nope.  We'll use whatever radix the address is in.